modules/gdscript/gd_functions.cpp

```diff
@@ -107,6 +108,8 @@ const char *GDFunctions::get_func_name(Function p_func) {
 		"Color8",
 		"print_stack",
 		"instance_from_id",
+		"rad2vec",
+		"deg2vec",
```

Comment:
That's incorrect -- those two should be right behind the name for "max" (otherwise, everything would go wild)